core/genesis.go

```diff
@@ -644,44 +646,72 @@ func DefaultHoodiGenesisBlock() *Genesis {
 	}
 }
 
+const (
+	devAccountCount      = 10
+	devAccountSeedPrefix = "dev-account-seed-"
+	devAccountBalance    = 10000
+)
+
+type DevAccount struct {
+	Address    common.Address
+	PrivateKey *ecdsa.PrivateKey
+	Balance    *big.Int
+}
+
+func GenerateDevAccounts() ([]DevAccount, error) {
```

Comment:
remove the publicly-exported type/function, and return the faucet address/key directly from `DeveloperGenesisBlock`.